MediaBrowser.Controller/Sorting/SortExtensions.cs

```diff
@@ -7,137 +7,25 @@ namespace MediaBrowser.Controller.Sorting
 {
     public static class SortExtensions
     {
+        private static IComparer<string> _comparer = new AlphanumComparator();
```

Comment:
```suggestion
        private static readonly AlphanumComparator<string> _comparer = new AlphanumComparator();
```
This doesn't need to be an interface, does it?

Reply:
Yeah, it doesn't. Though `AlphanumComparator` isn't a template class :)